turbinia/e2e/teardown-gce.sh

```diff
@@ -0,0 +1,36 @@
+#/bin/bash
+# This script will cleanup and teardown the Turbinia Terraform setup after
+# the Turbinia e2e test is finished.
+echo "Tear down Terraform Turbinia infrastructure."
+
+DEVSHELL_PROJECT_ID=`gcloud config list --format 'value(core.project)'`
+SA_MEMBER="terraform@$DEVSHELL_PROJECT_ID.iam.gserviceaccount.com"
+
+cd ./forseti-security/contrib/incident-response/infrastructure/
+terraform destroy --target=module.turbinia -var gcp_project=$DEVSHELL_PROJECT_ID -auto-approve
+
+# Remove test and evidence disks (test-disk2 and evidence* disks)
+ZONE="us-central1-a"
+gcloud -q compute disks delete test-disk2 --zone=$ZONE
```

Comment:
Should we have a config file that we store the project/zone data in?  It would be nice to explicitly list the project in the gcloud commands so that we can be sure we're operating on the correct project.  Eventually I'd like to be able to run e2e tests from my desktop, so everything should have safe defaults.

Reply:
I wonder if we can add some other validation here too?  Like maybe only deleting disks with 'turbinia' or 'test' in them here too?

Reply:
Currently *only* deleting the test disk (for which we know the exact name) and disks that start with "evidence" in the name.

Agree with having an option to set the zone/project from the CLI (maybe as a parameter to the command instead of a config file? (seems a bit overkill)) 

Will add the --project option to all gcloud commands, makes sense.